bower.json

```diff
@@ -28,6 +29,7 @@
     "underscore": "~1.8.3",
     "davclient.js": "https://github.com/evert/davclient.js.git",
     "es6-promise": "^3.2.1",
-    "ical.js": "^1.2.1"
+    "ical.js": "^1.2.1",
+    "web-pull-to-refresh": "git@github.com:ChristophWurst/web-pull-to-refresh.git#^1.1.0"
   }
```

Comment:
why should we if we can use bower to manage dependencies? That wouldn't solve any problem but make the install process for devs more complicated